### PR TITLE
Prep 2.1.0 Release

### PR DESCRIPTION
**Pull request by nmuesch**
Do the changelog for the 2.1.0 Release of our Terraform Provider.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
 ## 2.1.0 (Unreleased)
+
+FEATURES:
+* `datadog_dashboard`: New Resource combining screenbaord and timeboard, allowing a single config to manage all of your Datadog Dashboards. [GH-249]
```
Comment (bkabrda) [inline CHANGELOG.md]: Typo: screenbaord

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+* `datadog_integration_pagerduty_service_object`: New Resource that allows the configuration of individual pagerduty services for the Datadog Pagerduty Integration. [GH-237]
+
+IMPROVEMENTS:
+* `datadog_aws`: Add a mutex around all API operations for this resource. [GH-254]
+* `datadog_downtime`: General improvements around allowing the resource to be ran mutliple times without sending any unchanged values for the start/end times. Also fixes non empty diff when monitor_tags isn't set. [GH-264] [GH-267]
```
Comment (bkabrda) [inline CHANGELOG.md]: Typo: `mutliple`

File: CHANGELOG.md (hunk 1 above)
Comment (nmuesch) [reply CHANGELOG.md]: 🤦‍♂ My most common spelling mistake strikes again

File: CHANGELOG.md (hunk 2 above)
Comment (nmuesch) [reply CHANGELOG.md]: Thanks for the catch, fixed!

**Pull request merged**